sdk/compute/Azure.ResourceManager.Compute/src/autorest.md

```diff
@@ -181,121 +88,4 @@ directive:
   - from: swagger-document
     where: $.definitions.DiskSecurityType["x-ms-enum"].values[3]
     transform: $["name"] = "ConfidentialVmDiskEncryptedWithCustomerKey"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetVMProfile
-    transform: $["x-ms-client-name"] = "VirtualMachineScaleSetVmProfile"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetProperties.properties.doNotRunExtensionsOnOverprovisionedVMs
-    transform: $["x-ms-client-name"] = "DoNotRunExtensionsOnOverprovisionedVms"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetUpdateProperties.properties.doNotRunExtensionsOnOverprovisionedVMs
-    transform: $["x-ms-client-name"] = "DoNotRunExtensionsOnOverprovisionedVms"
-  - from: swagger-document
-    where: $.definitions.VMScaleSetConvertToSinglePlacementGroupInput
-    transform: $["x-ms-client-name"] = "VirtualMachineScaleSetConvertToSinglePlacementGroupOptions"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetVMInstanceIDs
-    transform: $["x-ms-client-name"] = "VirtualMachineScaleSetVmInstanceIds"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetVMInstanceRequiredIDs
-    transform: $["x-ms-client-name"] = "VirtualMachineScaleSetVmInstanceRequiredIds"
-  - from: swagger-document
-    where: $.paths["/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmScaleSetName}"].get.parameters[4]
-    transform: $["x-ms-enum"].name = "ExpandTypesForGetVirtualMachineScaleSets"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetVMInstanceView
-    transform: $["x-ms-client-name"] = "VirtualMachineScaleSetVmInstanceView"
-  - from: swagger-document
-    where: $.definitions.VirtualMachineScaleSetVMReimageParameters
-    transform: $["x-ms-client-name"] = "VirtualMachineScaleSetVmReimageOptions"
```

Comment:
Can we keep this renaming?

Reply:
These are the renaming of Update body parameter types. It should have been taken care of by recent changes in the generator, therefore we do not need them

Reply:
Actually it doesn't because it isn't an `Update` method, here is the [generated code](https://github.com/Azure/azure-sdk-for-net/blob/42669469ef8b37659e2d1019d25a23db8a43f110/sdk/compute/Azure.ResourceManager.Compute/src/Generated/VirtualMachineScaleSetVm.cs#L213) using latest autorest.csharp.

Reply:
Ah! Make sense. Let me fix this